derive/src/utils.rs

```diff
@@ -144,6 +172,12 @@ pub fn keccak(s: &str) -> H256 {
 	res
 }
 
+/// Returns the first 4 bytes of the Keccak hash (256-bits)
+/// for the given string slice.
+/// 
+/// # Note
+/// 
+/// The given string slice will be interpreted as bytes array.
```

Comment:
May I suggest renaming this function to something like `function selector`? `hash` is very ambiguous term, and [function selector](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI#function-selector) is more specific.

Reply:
Yeah I would also think that this is a better name for it. I will rename it.

Reply:
Renamed it, resolved.